### ISSUE TITLE
Media hotkeys do not work on MacOS

### ISSUE THREAD

**Issue by xkevinx** [jeffvli/sonixd#376]
**Describe the bug**

I made sure Sonixd was a trusted accessibility client, but so far the media hotkeys do not work on my Macbook. 

When I press the `play/pause` key, it opens Apple music or starts playing something I was playing before on my browser, but it never plays or pause anything on Sonixd.

**To Reproduce**
1. Play some music on Sonixd
2. Press the `play/pause` media hotkey
3. Nothing happens

**Expected behavior**
Music should pause

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS 12.5

**Additional context**
Nothing to add but I wanted to thank you for this cool app!

**Comment by xkevinx** [comment]
I am not sure if this will help, but I have noticed that if I turn off my bluetooth speakers, any music player would be paused by the system, but not sonixd. Perhaps what I'm going to say is total BS, but could it be that the player is not considered as a media or something?

**Comment by jeffvli** [comment]
Have you already added Sonixd as a [trusted accessibility client?](https://developer.apple.com/library/archive/documentation/Accessibility/Conceptual/AccessibilityMacOSX/OSXAXTestingApps.html) I've previously heard of some unresolved issues with bluetooth audio but media keys should work after adding it there.

**Comment by xkevinx** [comment]
Yes I do! So I have tried using the media hotkeys along the day today on my work computer, I managed to have them work a few times (around 10-20% of the times I tried) but most of the times they didn't work. There's definitely an issue there because I don't have any problem with other music apps.

**Comment by pztrn** [comment]
Remove sonixd from accessibility agents and re-add it (or just add it manually).

**Comment by xkevinx** [comment]
no sorry, it is not changing anything. Since the problem seems to be hard to replicate I'm fine with closing the issue, I can live without the media keys :)

**Comment by pztrn** [comment]
Probably stupid question, yet:

Is "Global Media Hotkeys" enabled in settings - player?

**Comment by bit101** [comment]
> 

I'm not the OP, but this was not a stupid question for me. :) I had reinstalled Sonixd. I knew enough to make sure it was trusted, but forgot that I had to toggle the setting. Thanks.

**Comment by xkevinx** [comment]
ahah yes could have happened to me but the setting was toggled already

**Comment by bit101** [comment]
oh, I read the initial description in more detail. This is a Mac OS "feature". I installed notunes which "fixes" this feature. https://github.com/tombonez/noTunes

**Comment by xkevinx** [comment]
@bit101 thanks for the advice! It does stop Apple Music to start when using the hotkeys, but they are still not doing any effect on Sonixd

**Comment by xkevinx** [comment]
ok I fixed it after digging a bit more online!

I went to Settings -> Accessibility -> Keyboard and I turned on `enable full keyboard access` and restarted the laptop. It looks all good now!

**Comment by ldexterldesign** [comment]
No solutions here worked for me except notunes, which suggests to me this is more of an overzealous Apple issue than a Sonixd one

How does everyone else's media keys behave with other apps?

Hope this helps
